frontend-auth-module/projects/sl-auth-app/src/assets/config/library-config.json

```diff
@@ -11,5 +11,7 @@
     "otpGenerationApiPathForResetPassword": "/elevate-user/v1/account/generateOtp",
     "supportEmail":"mentoredtest1@yopmail.com",
     "privacyPolicyUrl":"https://www.privacyploicy.com",
-    "termOfUseUrl":"https://www.termsofuse.com"
+    "termOfUseUrl":"https://www.termsofuse.com",
+    "intialPage": true,
```

Comment:
Check the spelling of initial in line 15 and 16

Reply:
done